react/components/ProductTitleAndPixel.tsx

```diff
@@ -38,91 +59,78 @@ interface Seller {
       return []
     }
 
-    const hasCategoryTree = categoryTree && categoryTree.length
-
-    const pageInfo: any = {
+    const pageInfo: PixelEvent = {
       event: 'pageInfo',
       eventType: 'productView',
       accountName: account,
-      pageCategory: 'Product',
-      pageDepartment: hasCategoryTree ? head(categoryTree!).name : '',
-      pageFacets: [],
       pageTitle: titleTag,
       pageUrl: window.location.href,
-      productBrandName: brand,
-      productCategoryId: Number(categoryId),
-      productCategoryName: hasCategoryTree ? last(categoryTree!).name : '',
-      productDepartmentId: hasCategoryTree ? head(categoryTree!).id : '',
-      productDepartmentName: hasCategoryTree ? head(categoryTree!).name : '',
-      productId: productId,
-      productName: productName,
-      skuStockOutFromProductDetail: [],
-      skuStockOutFromShelf: [],
     }
 
-    const { ean = undefined, referenceId = undefined, sellers = undefined } = selectedItem || {}
-
-    pageInfo.productEans = [ean]
-
-    if (referenceId && referenceId.length >= 0) {
-      const [{ Value: refIdValue }] = referenceId
-
-      pageInfo.productReferenceId = refIdValue
-    }
-
-    if (sellers && sellers.length >= 0) {
-      const [{ commertialOffer, sellerId }] = sellers
-
-      pageInfo.productListPriceFrom = `${commertialOffer.ListPrice}`
-      pageInfo.productListPriceTo = `${commertialOffer.ListPrice}`
-      pageInfo.productPriceFrom = `${commertialOffer.Price}`
-      pageInfo.productPriceTo = `${commertialOffer.Price}`
-      pageInfo.sellerId = `${sellerId}`
-      pageInfo.sellerIds = `${sellerId}`
-    }
-
-    const pageView = {
-      event: 'pageView',
-      pageTitle: titleTag,
-      pageUrl: location.href,
-      referrer:
-        document.referrer.indexOf(location.origin) === 0
-          ? undefined
-          : document.referrer,
-      accountName: account,
-    }
-
-    return [pageView, pageInfo]
-  }, [account, product, productName, selectedItem, titleTag])
+    return pageInfo
+  }, [account, titleTag])
 
   useDataPixel(pageEvents, path(['linkText'], product), loading)
 }
 
-function useProductEvents(product: Product, selectedItem: SKU, loading: boolean) {
+function getSkuProperties(item: SKU) {
+  return {
+    itemId: item.itemId,
+    name: item.name,
+    ean: item.ean,
+    referenceId: item.referenceId,
+    sellers: item.sellers.map(seller => ({
+      sellerId: seller.sellerId,
+      commertialOffer: {
+        Price: seller.commertialOffer.Price,
+        ListPrice: seller.commertialOffer.ListPrice,
+        AvailableQuantity: seller.commertialOffer.AvailableQuantity
+      }
+    }))
+  }
+}
+
+function useProductEvents(
+  product: Product,
+  selectedItem: SKU,
+  loading: boolean
+) {
   const productEvents = useMemo(() => {
-    if (!product || typeof document === 'undefined' || !selectedItem) {
+    const hasCategoryTree = product && product.categoryTree && product.categoryTree.length
```

Comment:
⛏[Nitpick] Make this explicitly a `boolean` by:
```tsx
const hasCategoryTree = Boolean(product && product.categoryTree && product.categoryTree.length)
```
Since even if this is truthy, it would be a `number`